tests/integration_test.rs

```diff
@@ -273,16 +266,6 @@ mod tests {
             response.key_id
         );
 
-        let auth_token = env::var("TEST_AUTH_TOKEN").expect("env var TEST_AUTH_TOKEN must be set");
-        let parts: Vec<&str> = auth_token.split('.').collect();
-        assert!(std::str::from_utf8(
-            base64_url::decode(parts[1])
-                .expect("expected base64 part not present in position 1")
-                .as_slice()
-        )
-        .expect("couldn't parse base64")
-        .contains(&response.endpoint));
```

Comment:
This was I think just some old code - v1 tokens are base64 encoded, so this was failing.